src/parallel.cpp

```diff
@@ -471,12 +468,12 @@ void CParallelValidation::HandleBlockMessage(CNode *pfrom,
             } // We must not hold the lock here because we could be waiting for a grant, below.
 
             // wait for semaphore grant
-            semPV->wait();
+            semaphore.wait();
```

Comment:
why change the name from semPV to semaphore?  in the codebase "sem" is the prefix for semaphores with the rest being a description of what it is...such as semOutbound for outbound connections...it's a little more descriptive and easier to read IMO.